data/registry/tools-cpp-alpine-apk.yml

```
@@ -0,0 +1,14 @@
+title: Alpine Linux Package for OpenTelemetry C++
+registryType: core
```

I see two options here. Either list all that apply, or we could use a convention like `*` or `mixed`:

```suggestion
registryType: core exporter
```

This doesn't break anything. In a followup PR we can adapt the filter to expect more than one keyword. WDYT?

---

makes sense :-)